### ISSUE TITLE
Investigate HELIDON_QUICKSTART_SE increased RSS in GHAs.

### ISSUE THREAD

**Issue by jerboaa** [Karm/mandrel-integration-tests#226]
We apparently see a `~2MB` increase in RSS size in GHA:

```
 Error:    RuntimesSmokeTest.helidonQuickStart:228->testRuntime:77->testRuntime:163 
Application HELIDON_QUICKSTART_SE in mode none consumed 78764 kB of RSS memory , which is over 78368 kB threshold by 1%.
 ==> expected: <true> but was: <false>
```

The reference value for 23.1 is: 76647 Kb This needs to be investigated.

See:
https://github.com/Karm/mandrel-integration-tests/issues/198#issuecomment-1825698339

**Comment by jerboaa** [comment]
CI failure is here: https://github.com/graalvm/mandrel/actions/runs/6968236452/job/18962635519#step:11:12851

**Comment by jerboaa** [comment]
There has been a threshold bump in https://github.com/Karm/mandrel-integration-tests/pull/227 but we need to do the threshold handling per quarkus version probably.

**Comment by Karm** [comment]
@jerboaa Helidon has nothing to do with Quarkus. Our Helidon version is very old, the last time we updated Helidon was in April 2021: https://github.com/Karm/mandrel-integration-tests/commit/9cab8f92186e975d8f807a56e7844fcb893180fe

If the RSS is off, it's either:
* environment, OS
* or Mandrel

**Comment by Karm** [comment]
> Which confirms it needs some investigation?

Indeed. A rare occasion where it seems we have fewer moving parts as the app version is the same.

**Comment by zakkak** [comment]
> Can we disable this test again - until we can reliably enable it again - as it's causing a lot of noise?

Done in https://github.com/graalvm/mandrel/pull/626

**Comment by jerboaa** [comment]
Thanks!

**Comment by zakkak** [comment]
> Interestingly, the only time it passed was in https://github.com/zakkak/mandrel/actions/runs/6944009708/job/18891435883 (while testing before merging the PR that enabled these tests)

Turns out that's not true.

The test was passing in https://github.com/graalvm/mandrel/actions/runs/6999473951/job/19039243355#step:11:13026

So the first known bad commit seems to be 61edae46e0287ecaee7f5b3ef728ab4a6d465c02 in https://github.com/graalvm/mandrel/actions/runs/7026828114/job/19120880487

The last known working commit seems to be 670b43cac8587d27d7810af41c22ca3cf161df23 in https://github.com/graalvm/mandrel/actions/runs/7012992753/job/19078995046
